alpine/LandauDampingManager.h

```diff
@@ -116,116 +112,118 @@ class LandauDampingManager
     }
     void post_step() override {
         // Update time
-        this->time_m += this->dt;
-        this->it++;
+        time_m += dt_m;
+        it_m++;
         // wrtie solution to output file
-        this->dump();
+        dump();
 
         Inform m("Post-step:");
-        m << "Finished time step: " << this->it << " time: " << this->time_m << endl;
+        m << "Finished time step: " << it_m << " time: " << time_m << endl;
     }
     void pre_run() override {
         Inform m("Pre Run");
         for (unsigned i = 0; i < Dim; i++) {
-            this->domain[i] = ippl::Index(this->nr[i]);
+            domain_m[i] = ippl::Index(nr_m[i]);
         }
 
-        this->decomp.fill(true);
-        this->kw    = 0.5;
-        this->alpha = 0.05;
-        this->rmin  = 0.0;
-        this->rmax  = 2 * pi / this->kw;
+        decomp_m.fill(true);
+        kw_m    = 0.5;
+        alpha_m = 0.05;
+        rmin_m  = 0.0;
+        rmax_m  = 2 * pi / kw_m;
 
-        this->hr = this->rmax / this->nr;
+        hr_m = rmax_m / nr_m;
         // Q = -\int\int f dx dv
-        this->Q = std::reduce(this->rmax.begin(), this->rmax.end(), -1., std::multiplies<double>());
-        this->origin = this->rmin;
-        this->dt     = std::min(.05, 0.5 * *std::min_element(this->hr.begin(), this->hr.end()));
-        this->it     = 0;
+        Q_m = std::reduce(rmax_m.begin(), rmax_m.end(), -1., std::multiplies<double>());
+        origin_m = rmin_m;
+        dt_m     = std::min(.05, 0.5 * *std::min_element(hr_m.begin(), hr_m.end()));
+        it_m     = 0;
+        time_m   = 0.0;
 
         m << "Discretization:" << endl
-          << "nt " << this->nt << " Np= " << this->totalP << " grid = " << this->nr << endl;
+          << "nt " << nt_m << " Np= " << totalP_m << " grid = " << nr_m << endl;
 
-        std::shared_ptr<Mesh_t<Dim>> mesh = std::make_shared<Mesh_t<Dim>>(this->domain, this->hr, this->origin);
+        std::shared_ptr<Mesh_t<Dim>> mesh = std::make_shared<Mesh_t<Dim>>(domain_m, hr_m, origin_m);
 
-        this->isAllPeriodic = true;
+        isAllPeriodic_m = true;
 
-        std::shared_ptr<FieldLayout_t<Dim>> FL = std::make_shared<FieldLayout_t<Dim>>(MPI_COMM_WORLD, this->domain, this->decomp, this->isAllPeriodic);
+        std::shared_ptr<FieldLayout_t<Dim>> FL = std::make_shared<FieldLayout_t<Dim>>(MPI_COMM_WORLD, domain_m, decomp_m, isAllPeriodic_m);
```

Comment:
Same here. Why do you need them as a pointer?

Reply:
see above